src/logics/LIALogic.cc

```diff
@@ -188,9 +175,10 @@ opensmt::pair<FastRational, PTRef> LIALogic::sumToNormalizedPair(PTRef sum) {
     assert(std::all_of(coeffs.begin(), coeffs.end(), [](opensmt::Number const & coeff) { return coeff.isInteger(); }));
     // Now make sure all coeffs are coprime
     auto coeffs_gcd = abs(coeffs[0]);
-    for (std::size_t i = 1; i < coeffs.size() && coeffs_gcd != 1; ++i) {
-        coeffs_gcd = gcd(coeffs_gcd, abs(coeffs[i]));
+    for (auto const & coeff : coeffs) {
+        coeffs_gcd = gcd(coeffs_gcd, abs(coeff));
```

Comment:
This now adds one unnecessary `gcd` operation of the first coefficient with itself.
I think I would rather stick to the original version.

Reply:
Right, it slipped past me, I didn't want to commit it.  I'll take it off.

Reply:
OK. Otherwise the refactoring is nice!